modules/scrabble.js

```diff
@@ -55,5 +55,19 @@ module.exports = {
         return String(scrabbleScore(msg.args.word) || 'Not a valid word')
       }
     }
+  },
+  events: {
+    message: function (bot, nick, to, text) {
+      var wordScores = {}
+      for (const word of text.split(' ')) {
```

Comment:
`text.split()` forms an array, so you could `.map()` if you wanted
(but this way is fine)

Reply:
oh it's an object, maybe not